public/main.js

```
@@ -4,135 +4,140 @@
-			break;
-		case "playerId":
-			console.debug(msgJson.text);
-			localStorage.setItem ("playerId", msgJson.text);
-			break;			
-	}
+document.getElementById("rollDices").addEventListener("click", function () {
+  connection.send(message("roll"));
+});
 
+connection.setOnmessage(function (event) {
+  console.debug(msgJson.type);
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)